src/d/d_stage.cpp

```diff
@@ -1315,10 +1315,10 @@ int dStage_cameraCreate(stage_camera2_data_class* i_cameraData, int i_cameraIdx,
     i_cameraData = static_cast<stage_camera2_data_class*>(cMl::memalignB(-4, 0x18));
 
     if (i_cameraData != NULL) {
-        i_cameraData->field_0x4 = 0.0f;
-        i_cameraData->field_0x8 = 0.0f;
-        i_cameraData->field_0x4 = 0.0f;
-        i_cameraData->field_0x0 = param_2;
+        i_cameraData->m00.mpTypeStr[1] = 0;
+        i_cameraData->m00.mpTypeStr[2] = 0;
+        i_cameraData->m00.mpTypeStr[1] = 0;
+        i_cameraData->m00.mpTypeStr[2] = param_2;
```

Comment:
This isn't right and caused the function's match% to regress:
`dStage_cameraCreate__FP24stage_camera2_data_classii | fuzzy_match  | 100.00% -> 81.48%`

I'm gonna need to clean these structs up later since their names were already wrong, but for now let's just revert the camera-related structs in d_stage.h to how they previously were (correct contents but wrong names) so that they at least don't get worse